app/views/contestants/index.html.erb

```diff
@@ -0,0 +1,7 @@
+<h1> Contestants </h1>
+<% @contestants.each do |contestant| %>
+  <p><%= contestant.name %></p>
+  <% contestant.contestant_projects.each do |project| %>
+    <p><%= project.project.name %></p>
```

Comment:
This works but it's a little bit more than is really needed - you're looping through this contestant's `contestant_projects`; but remember that a contestant also has _many projects_ thanks to its association _through_ the join table. So, this could really look like this instead, and read a lot better: 

```
contestant.projects.each do |project|
   <p><%= project.name %></p>
end
```

Reply:
This way it's referencing the project object directly, instead of one layer deeper.